apps/rig_inbound_gateway/lib/rig_inbound_gateway/api_proxy/handler/http.ex

```diff
@@ -84,12 +106,22 @@ defmodule RigInboundGateway.ApiProxy.Handler.Http do
 
     receive do
       {:response_received, response} ->
+        ProxyMetrics.count_proxy_request(conn.method, request_path, "http", response_from, "ok")
+
         conn
         |> with_cors()
         |> Conn.put_resp_content_type("application/json")
         |> Conn.send_resp(:ok, response)
     after
       response_timeout ->
+        ProxyMetrics.count_proxy_request(
+          conn.method,
+          request_path,
+          "http",
+          response_from,
+          "gateway_timeout"
```

Comment:
perhaps it would make sense to make a distinction between a gateway timeout with respect to the forwarded http request, and a timeout that happens while waiting for the response..

Reply:
It's `bad_gateway `& `gateway_timeout`. Isn't it?

Reply:
fixed as discussed offline